libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dependentFilters.spec.ts

```diff
@@ -36,7 +36,11 @@ describe("Dependent filter", () => {
 
         stateFilter.isLoaded().open().selectAttribute(["Connecticut"]).apply();
 
-        table.waitLoaded().getColumnValues(2).should("deep.equal", ["Bridgeport", "Hartford"]);
+        table
+            .waitLoadStarted()
```

Comment:
we usually have bad experience with these, sometimes the loading finishes even before cypress starts waiting for it and then waits forever. As we have no better thing now (we could try later race(fixedTime, waitLoadingStarted), keep as is and we can adjust later in case of issues.